.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true
+  },
+  extends: 'standard',
```

Comment:
Use `eslint` tool with standard style guide in favor of maintaining custom lint configs.